vitrina/smart_contracts/models.py

```diff
@@ -19,3 +20,74 @@ class SmartContractTemplate(UUIDBaseModel):
         null=True,
         blank=True,
     )
+
+
+class Agreement(UUIDBaseModel):
+    CREATED = "CREATED"
+    FORMED = "FORMED"
+    INITIATED = "INITIATED"
+    SIGNED = "SIGNED"
+    ACTIVE = "ACTIVE"
+    DEACTIVATED = "DEACTIVATED"
+
+    STATUSES = (
```

Comment:
optional: textchoices can be used: https://docs.djangoproject.com/en/5.2/ref/models/fields/#enumeration-types